src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -118,18 +123,31 @@ public CallbackService(final SecurityUtils securityUtils,
         HttpHeaders securityHeaders = securityUtils.authorizationHeaders();
 
         CallbackTelemetryThreadContext.setTelemetryContext(new CallbackTelemetryContext(callbackType));
+        List<String> ccdCallbackLogControl = applicationParams.getCcdCallbackLogControl();
+        boolean match = false;
+        String starStr = "*";//to match any call back
```

Comment:
this should be a private static final, all upper case

Reply:
refactored